Automattic/wp-calypso
# Create Happiness Engineers Tray Component

#### Changes proposed in this Pull Request

This is the first step to #38998. With that, the row of Happiness Engineers' Gravatars would be used three times throughout Calypso, with each use case requiring the same implementation and styling. It probably makes sense to just create a component for it. 

#### Testing instructions

There's two cases to test. Firstly, check `/hosting-config/site`, but remove the `feature-example__gradient` class if you don't have a Business Plan.

<img width="1244" alt="Screenshot 2020-02-29 at 20 17 11" src="https://user-images.githubusercontent.com/43215253/75614593-4c035280-5b32-11ea-903e-822c7818d855.png">

Secondly, check the bottom of `/help`
<img width="1070" alt="Screenshot 2020-02-29 at 20 17 18" src="https://user-images.githubusercontent.com/43215253/75614596-4efe4300-5b32-11ea-8266-efcfda5917d3.png">

The third case will be with #38998.

cc @mmtr @gwwar
